release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -81,6 +81,15 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-oc"]
 === OpenShift CLI (oc)
+==== New flag added to run must-gather in a specified namespace
```

Comment:
This needs an ID above it. It would look something like:

```suggestion
[id="ocp-4-13-oc-must-gather-namespace"]
==== New flag added to run must-gather in a specified namespace
```

(But you can name the ID however you want)

Reply:
Done.